run.sh

```diff
@@ -134,11 +134,17 @@ fi
 
 ## DNS Timers
 if [ ! -z "${NETWORK_ADDRESS_CACHE_TTL}" ]; then
-    sed -i -e "s/#networkaddress.cache.ttl=.*/networkaddress.cache.ttl=${NETWORK_ADDRESS_CACHE_TTL}/" /opt/jdk-*/conf/security/java.security
+    sed -i -e "s/#networkaddress.cache.ttl=.*/networkaddress.cache.ttl=${NETWORK_ADDRESS_CACHE_TTL}/" /usr/share/elasticsearch/jdk/conf/security/java.security
```

Comment:
this is now set in jvm.options ... so we should just set it there and stop overriding in the entrypoint ... which also mean soon we might just be able to get rid of this script ( the upstream image must do something about `initial_master_nodes` as well ? )

*jvm.options*
```
## DNS cache policy
# cache ttl in seconds for positive DNS lookups noting that this overrides the
# JDK security property networkaddress.cache.ttl; set to -1 to cache forever
-Des.networkaddress.cache.ttl=60
# cache ttl in seconds for negative DNS lookups noting that this overrides the
# JDK security property networkaddress.cache.negative ttl; set to -1 to cache
# forever
-Des.networkaddress.cache.negative.ttl=10
```

Reply:
Moved to jvm.options.